include/vcpkg/paragraphs.h

```diff
@@ -22,11 +26,20 @@ namespace vcpkg::Paragraphs
 
     bool is_port_directory(const ReadOnlyFilesystem& fs, const Path& maybe_directory);
 
-    ParseExpected<SourceControlFile> try_load_port(const ReadOnlyFilesystem& fs, const Path& port_directory);
-    ParseExpected<SourceControlFile> try_load_port_text(const std::string& text,
-                                                        StringView origin,
-                                                        bool is_manifest,
-                                                        MessageSink& warning_sink);
+    // If an error occurs, the Expected will be in the error state.
+    // Otherwise, if the port is known, the Optional contains the loaded port information.
+    // Otherwise, the Optional is disengaged.
```

Comment:
```suggestion
    // Otherwise, if the port is known, the unique_ptr contains the loaded port information.
    // Otherwise, the unique_ptr is nullptr.
```